ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -52,10 +112,7 @@ export default function TokenList({
   };
 
   const sortedTokens = useMemo(() => {
-    return sortAssets(
-      [nativeTokenWithBalance, ...tokensWithBalances],
-      tokenSortConfig,
-    );
```

Comment:
Why are we removing native tokens from the sorting functionality here? We will need to include the native tokens with their balances, along with all erc20's so that they can be sorted in the correct order by declining fiat balance.

Reply:
This is a wip, I removed it to simplify my testing. I'm looking at hardcoding marketData now